src/scales/scale.ts

```diff
@@ -105,6 +105,15 @@ export class Scale<D, R> {
   }
 
   /**
+   * Gets an array of tick values spanning the domain.
+   *
+   * @returns {D[]}
+   */
+  public ticks(): D[] {
+    return this.domain();
```

Comment:
I like the shared api of `ticks` better than type checking in the gridlines code.